src/Symfony/Component/Debug/DebugClassLoader.php

```diff
@@ -307,6 +307,8 @@ public function checkAnnotations(\ReflectionClass $refl, $class)
                     }
                 }
             }
+
+            unset($areFromTheSameVendor);
```

Comment:
I'd suggest removing this line and adding `$areFromTheSameVendor = null` before L279